Formula/fluid-synth@2.1.rb

```diff
@@ -17,6 +17,8 @@ class FluidSynthAT21 < Formula
 
   keg_only :versioned_formula
 
+  deprecate! date: "2022-08-15", because: :versioned_formula
```

Comment:
No dedicated upstream branch. There was a single release after 2.2.0 with backported fix on 2021-05-08 (https://github.com/FluidSynth/fluidsynth/releases/tag/v2.1.9). Since nothing else uses this anymore, might as well deprecate now.